rules/manager.go

```diff
@@ -49,7 +45,7 @@ var (
 			Name:      "rule_evaluation_duration_seconds",
 			Help:      "The duration for a rule to execute.",
 		},
-		[]string{ruleTypeLabel},
+		[]string{"rule_type", "status"},
```

Comment:
Well @grobie was asking about it yesterday. I don't find it entirely unreasonable.
Another option would be not observing failures – but since failures can be due to timeouts, it's relevant I would say.